src/OpenTelemetry/Metrics/Exemplar/ExemplarFilterType.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+#if EXPOSE_EXPERIMENTAL_FEATURES && NET8_0_OR_GREATER
+using System.Diagnostics.CodeAnalysis;
+using OpenTelemetry.Internal;
+#endif
+
+namespace OpenTelemetry.Metrics;
+
+#if EXPOSE_EXPERIMENTAL_FEATURES
+/// <summary>
+/// Defines the exemplar filters supported by the OpenTelemetry .NET Metrics
+/// SDK.
+/// </summary>
+/// <remarks>
+/// <inheritdoc cref="Exemplar" path="/remarks/para[@experimental-warning='true']"/>
+/// Specification: <see
+/// href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#exemplarfilter"/>.
+/// </remarks>
+#if NET8_0_OR_GREATER
+[Experimental(DiagnosticDefinitions.ExemplarExperimentalApi, UrlFormat = DiagnosticDefinitions.ExperimentalApiUrlFormat)]
+#endif
+public
+#else
+internal
+#endif
+    enum ExemplarFilterType
+{
+    /// <summary>
+    /// An exemplar filter which makes no measurements eligible for becoming an
+    /// <see cref="Exemplar"/>.
+    /// </summary>
+    /// <remarks>
+    /// Specification: <see
+    /// href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#alwaysoff"/>.
+    /// </remarks>
+    AlwaysOff,
+
+    /// <summary>
+    /// An exemplar filter which makes all measurements eligible for becoming an
+    /// <see cref="Exemplar"/>.
+    /// </summary>
+    /// <remarks>
+    /// Specification: <see
+    /// href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#alwayson"/>.
+    /// </remarks>
+    AlwaysOn,
+
+    /// <summary>
+    /// An exemplar filter which makes measurements recorded in the context of a
+    /// sampled <see cref="Activity"/> (span) eligible for becoming an <see
```

Comment:
```suggestion
    /// sampled parent <see cref="Activity"/> (span) eligible for becoming an <see
```
to match the spec definition.

Reply:
This "parent" text was in a couple spots. I removed it because I thought it was super confusing!

> An ExemplarFilter which makes those measurements eligible for being an Exemplar, which are recorded in the context of a sampled parent span.

Does that mean `Activity.Current.Parent.Recorded` or `Activity.Current.Recorded`?

> An ExemplarFilter which makes those measurements eligible for being an Exemplar, which are recorded in the context of a sampled span.

Seems to more clearly mean `Activity.Current.Recorded`?

Reply:
I think the `parent` term is used with respect to the measurement here. But I see the confusion, ok to keep what you have.